### ISSUE TITLE
Prioritize transport aircraft for troop transport from one CP to any other CP that is further than the closest

### ISSUE THREAD

**Issue by Hornet2041** [dcs-liberation/dcs_liberation#1977]
discord discussion starts here:

https://discord.com/channels/595702951800995872/595706061126631424/932403103918084168

tl;dr in big campaigns, transporting a lot of troops from far away factory CPs can take quite a few turns to make it to the frontline. What if, with transport aircraft inventory permitting and the airspace between the two friendly CPs is clear of threats, the autoplanner would instead send purchased units via transport aircraft?

Since large airplane transports can only take 2 units max with Liberation currently not making any differences between the capacity of actual RL specifications, I was thinking the autoplanner could prioritize by unit cost, as this would mean tanks would usually get selected first to fly and then the rest of the ~~trash~~ ground units could take the highways like chumps.

This could also generate more flight options such as ESCORT to ensure enemy fighters don't sneak in to shoot them down or regular INTERDICTION missions to shoot down aerial convoys.

For helicopters, since they can only take one (but you can buy and store way more), I was thinking the autoplanner would use them for FOB - to - FOB transfers, but that one is still kinda meh for me...

**Comment by DanAlbert** [comment]
This was an intentional design decision to ensure that convoy interception missions could be planned often.

**Comment by Hornet2041** [comment]
sadness
